src/validator/ValidatorClient.ts

```diff
@@ -206,16 +213,19 @@ export function setIgnore(responseId: string): Promise<string[]> {
   return request(body) as Promise<string[]>;
 }
 
-export function closeValidator() {
+export function closeValidator(): Promise<unknown> {
+  console.log("Validator is closing");
   if (validationProcess) {
     const method = "shutdown";
     const body = {
       method: method,
+      params: {},
     };
     const promise = request(body) as Promise<string[]>;
     validationProcess.shutdowned = true;
     return promise;
   }
+  return Promise.resolve();
```

Comment:
why?

Reply:
To make the return type constant (Promise<unknown>). Remove this?

Reply:
OK